core/clingutils/res/TClingUtils.h

```diff
@@ -285,6 +302,12 @@ class AnnotatedRecordDecl {
       if (fRequestNoInputOperator) result |= kNoInputOperator;
       if (fRequestStreamerInfo) result |= kStreamerInfo;
       if (fRequestedVersionNumber > -1) result |= kHasVersion;
+      switch (fRequestedRNTupleSplitMode) {
+      case 0: break;
+      case 1: result |= kNtplForceSplit; break;
+      case -1: result |= kNtplForceUnsplit; break;
+      default: assert(false && "invalid setting of fRequestedRNTupleSplitMode");
+      }
```

Comment:
```suggestion
      switch (fRequestedRNTupleSplitMode) {
         case 0: break;
         case 1: result |= kNtplForceSplit; break;
         case -1: result |= kNtplForceUnsplit; break;
         default: assert(false && "invalid setting of fRequestedRNTupleSplitMode");
      }
```

Reply:
I find this nicer, too, but our `clang-format` currently doesn't...